docs/flipflop/jk_flipflop.md

```diff
@@ -21,7 +21,7 @@ JK flip-flop is the modified version of SR flip-flop. It operates with only posi
 
 This circuit has two inputs J & K and two outputs Q(t) & Q(t)’. The operation of JK flip-flop is similar to SR flip-flop. Here, the inputs of SR flip-flop are considered as **S = J Q(t)’** and **R = KQ(t)** in order to utilize the modified SR flip-flop for 4 combinations of inputs.
 
-## State table of **SR** flip-flop
+## State table of JK flip-flop
```

Comment:
SR

Reply:
But the state table is of JK, and it is the `jk_flipflop` section.

Reply:
Oh my bad sweat_smile.